diff --git a/network/services/quickstart/Makefile b/network/services/quickstart/Makefile
index 773f763..3e5d4ad 100644
--- a/network/services/quickstart/Makefile
+++ b/network/services/quickstart/Makefile
@@ -29,7 +29,7 @@ define Package/$(PKG_NAME)
 	SUBMENU:=Web Servers/Proxies
 	TITLE:=Quickstart, the quick start.
 	DEPENDS:=@(x86_64||aarch64||arm) +shadow-utils +shadow-useradd +mount-utils +block-mount +lsblk +e2fsprogs +parted \
-		+smartmontools +smartd +smartmontools-drivedb +mdadm +ip-full +ubus +uci +bash
+		+smartmontools +smartd +smartmontools-drivedb +ip-full +ubus +uci +bash
 	URL:=https://easepi.linkease.com/
 endef
 
